self-service-kiosk/src/index.js

```diff
@@ -3,16 +3,28 @@ import {createRoot} from 'react-dom/client';
 import {Provider} from 'react-redux';
 import {store} from './app/store/store.js';
 import './index.css';
-import {Test} from "./app/views/test.js";
+import { CustomerDisplay } from "./app/views/customer-display/customer-display";
 
+import {BrowserRouter, Route, Routes} from "react-router-dom";
+import {Helmet} from "react-helmet";
 
 
 const container = document.getElementById('root');
 const root = createRoot(container);
 
 root.render(
     <Provider store={store}>
-        <Test/>
+
+        <Helmet>
```

Comment:
ca fait une sorte de titre? c'est ca?

Reply:
It allows to update the `<head>` part (title, metadata...)